docs/user-guide/workflow.ipynb

```diff
@@ -138,9 +138,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "he3_workflow = pol.He3CellWorkflow()\n",
+    "he3_polarizer_workflow = pol.He3CellWorkflow(in_situ=True, incoming_polarized=False)\n",
```

Comment:
Unused?

Reply:
Yes right. I anyhow wanted to add it to provide the full overview of possibilities for the Full pipeline. I will add a text for that and then adapt whats used for the image